index.html

```diff
@@ -8,18 +8,36 @@
 		<link rel="stylesheet" href="style.css" />
 	</head>
 	<body>
-		<div class="centered">
-			<h1 id="text">Hello Git</h1>
-			<button type="button" onclick="changeColor()">Change color</button>
-			<p>
-				Lorem, ipsum dolor sit amet consectetur adipisicing elit.
-				Voluptates dolore dignissimos recusandae esse in cum
-				reprehenderit eveniet dolorem itaque? Accusantium quo saepe sunt
-				magnam voluptas explicabo laboriosam quidem repellat enim.
-			</p>
+		<header>
+			<h1>Header</h1>
+		</header>
+		<div class="home">
+			<section class="home-slidebar">
+				<h1>Slidebar 1</h1>
+			</section>
+			<section class="home-article">
```

Comment:
You name your class `home-article`, why don't you use the HTML 5 element `<article>` then?